news/2024/2024-09-17-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,175 @@
+- [Givikap120](https://osu.ppy.sh/users/10560705) refactored and cleaned up [various](https://github.com/ppy/osu/pull/29293) [areas](https://github.com/ppy/osu/pull/29294) [of](https://github.com/ppy/osu/pull/29292) [performance calculation code](https://github.com/ppy/osu/pull/29291) to make them easier to understand and maintain.
+- An [aim skill multiplier rebalance](https://github.com/ppy/osu/pull/29998) proposed by [tsunyoku](https://osu.ppy.sh/users/11315329) was made in order to ensure the final results of all major changes are in line with expectations.
+- A [speed accuracy scaling change](https://github.com/ppy/osu/pull/30088) proposed by [StanR](https://osu.ppy.sh/users/7217455) was made to punish lower accuracies below OD 8 more harshly.
+- [Finadoggie](https://osu.ppy.sh/users/14182048)'s proposed [score statistics estimation change](https://github.com/ppy/osu/pull/27691) was made to ensure that slider breaks and dropped slider ends are correctly accounted for in osu!(lazer) scores.
+
+## osu!taiko
+
+In order to aid understanding of the changes to osu!taiko, these are the skills in difficulty calculation which will be referenced throughout:
+
+- **Stamina**: the speed at which you hit notes, based on an assumed finger count of 2 per colour
+- **Colour**: the frequency of which the beatmap changes between a don or kat
+- **Rhythm**: the complexity of the beatmap's rhythm in relation to notes' independent rhythm ratios
+
+### Consideration for TL tapping in the stamina component of difficulty calculation
+
+With the recent update to osu!taiko's difficulty calculation, an issue has arisen concerning a unique playstyle known as TL-tapping. [Maxie's video](https://www.youtube.com/watch?v=WM7gkFBDl9s) provides a more in-depth explanation of this playstyle.
+
+A recent [change](https://github.com/ppy/osu/pull/20558) by [vun](https://osu.ppy.sh/users/6932501) addresses these issues within the stamina system. When the new stamina system for osu!taiko was introduced in 2022, it assumed that players used two fingers per colour, leading to some impressive plays by players like [Ney](https://osu.ppy.sh/users/5991961) on [Alive](https://osu.ppy.sh/scores/1873110816) and others on [StrangeProgram](https://osu.ppy.sh/beatmapsets/5774#taiko/28065), which became notorious for yielding some speedy high PP scores.
+
+The new system, while simple in its implementation, introduces variable finger counts for stamina. Mono-colour patterns that last more than 300ms without a colour change are now considered to have four fingers available. This adjustment significantly nerfs certain maps and converts. Additionally, convert-specific nerfs have been removed from difficulty calculations to ensure that converts are weighted fairly against mode-specific maps. The changes to finger count availability also ensure that more *niche* skill sets are fairly represented.
```

Comment:
what "niche" skill sets? is it just the aforementioned tl-tapping method?

Reply:
@Lawtrohux

Reply:
not ltca but i think the intention was to include other playstyles which may use 3 or 4 fingers per color in the phrasing, nothing specific

Reply:
could we then reword to something along the lines of `The changes to finger count availability also ensure that play styles using varying amounts of fingers per colour are fairly represented`?

Reply:
i say go for it

Reply:
yeah that sounds fine to me. essentially yes its skillsets that utilise more than the set 2 finger count

Reply:
8446862a37994c10146eefb78835d72feb646b0f